<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditView</class>
 <widget class="QDialog" name="CreditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="horizontalSpacing">
    <number>-1</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="8" column="2">
    <widget class="QRadioButton" name="radioA">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Аннуитетные</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="calc">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QRadioButton" name="radioD">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Дифференцированные</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3" rowspan="4" colspan="2">
    <widget class="QCustomPlot" name="graf" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QCustomPlot{
	border-top: 0px solid rgb(55, 55, 55);
}</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_rate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Процентная ставка:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_amount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Сумма кредита:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" alignment="Qt::AlignVCenter">
    <widget class="QLineEdit" name="line_r">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	font: 13pt &quot;Times New Roman&quot;;
	color: rgb(255, 255, 255);
    background-color: rgb(225, 116, 104);
    border: 2px solid rgb(55, 55, 55);
    border-top: 0px solid rgb(55, 55, 55);
}</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_type">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Тип ежемесячных платежей:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" alignment="Qt::AlignVCenter">
    <widget class="QLineEdit" name="line_term">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	font: 13pt &quot;Times New Roman&quot;;
	color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
    border: 2px solid rgb(55, 55, 55);
	border-bottom: 0px solid rgb(55, 55, 55);
	border-top: 0px solid rgb(55, 55, 55);
}</string>
     </property>
     <property name="maxLength">
      <number>100</number>
     </property>
     <property name="placeholderText">
      <string>Срок кредита</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_term">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Срок кредита:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" alignment="Qt::AlignVCenter">
    <widget class="QLineEdit" name="line_amount">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	font: 13pt &quot;Times New Roman&quot;;
	color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
    border: 2px solid rgb(55, 55, 55);
	border-bottom: 0px solid rgb(55, 55, 55);
}</string>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Сумма кредита</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_percent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" alignment="Qt::AlignVCenter">
    <widget class="QLineEdit" name="line_t">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	font: 13pt &quot;Times New Roman&quot;;
	color: rgb(255, 255, 255);
     background-color: rgb(225, 116, 104);
     border: 2px solid rgb(55, 55, 55);
	border-top: 0px solid rgb(55, 55, 55);
}</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <widget class="QTableWidget" name="table">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget  {
	font: 13pt &quot;Times New Roman&quot;;
	color: rgb(60, 59, 60);
    background-color: rgb(255, 255, 255);
    border: 2px solid rgb(55, 55, 55);
}</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="box_term">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>лет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>месяцев</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="line_rate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	font: 13pt &quot;Times New Roman&quot;;
	color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
	border: 2px solid rgb(55, 55, 55);
	border-bottom: 0px solid rgb(55, 55, 55);
}</string>
     </property>
     <property name="placeholderText">
      <string>Процентная ставка</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
